pkg/core/label_list.go

```diff
@@ -133,7 +139,23 @@ func listLabelsChan(repo string, stores context2.Stores, prefix string, opts ...
 	keysChan := make(chan keyBatchEvent, 1)
 
 	iterator := func(next string) ([]string, string, error) {
-		return GetLabelStore(stores).KeysPrefix(context.Background(), next, model.GetArchivePathPrefixToLabels(repo, prefix), "", settings.batchSize)
+		if settings.label != "" {
+			return GetLabelStore(stores).KeysPrefix(context.Background(), next,
+				model.GetArchivePathPrefixToLabels(repo, prefix),
+				"", settings.batchSize)
+		} else {
+			// tbd: sandbox o.(type).method() type assert fail compared to o.(type).method() error
```

Comment:
there is nothing "tbd" left here. No need to comment on it

Reply:
i intended to throw together an example to broadcast in Slack#engineering. not sure if there's yet time for that now.  that's what this comment was about, tho.